src/core/public/styles/_base.scss

```diff
@@ -26,6 +26,10 @@
   margin-left: 320px; // Hard-coded for now -- @cchaos
 }
 
+.euiBody--hasFlyout[style*='padding-left: 248px'] .euiBottomBar {
+  margin-left: 248px; // Hard-coded for serverless sideNav
+}
```

Comment:
If we go that way, then I think we need some kind of automated test to make sure the layout stays fixed (e.g the day we use switch the padding from 248px to 249px)

Reply:
@pgayvallet any clue on where will be the good place to add this kind of test?

Reply:
@pgayvallet I was thinking of adding a test with a comment that assert nav size here https://github.com/nchaulet/kibana/blob/04df3f3ed939b942bd3bbdd6d7630321e069b6fb/x-pack/test_serverless/functional/test_suites/common/home_page.ts#L20 does it make sense to you?

Reply:
Yeah it would be fine ihmo.

Reply:
Just added a test in https://github.com/elastic/kibana/pull/166840/commits/11a4e7eadca6b4d5c44ff0c7111fd0a71a0485d7